src/commands.js

```diff
@@ -55,3 +54,6 @@ module.exports.getCommandChannels = function(client)
 
   return commandChannels;
 }
+module.exports.commandChannels = getCommandChannels
```

Comment:
Maybe I'm dumb, but why isn't it `module.exports.commandChannels = getCommandChannels();`